packages/api/lambdas/sf-starter.js

```diff
@@ -139,20 +139,23 @@ function handleThrottledEvent(event, visibilityTimeout) {
 
 async function handleSourceMappingEvent(event) {
   const sqsRecords = event.Records;
-  return await Promise.all(sqsRecords.map(
-    (sqsRecord) => dispatch(sqsRecord.eventSourceARN, sqsRecord)
-  ));
-}
-
-/**
- * Handler for messages from normal SQS queues.
- *
- * @param {Object} event - Lambda input message from SQS
- * @returns {Promise} - A promise resolving to how many executions were started
- * @throws {Error}
- */
-async function sqs2sfHandler(event) {
-  return await handleEvent(event, dispatch);
+  const batchItemFailures = [];
+  await Promise.all(sqsRecords.map(async (sqsRecord) => {
+    try {
+      return await dispatch(sqsRecord.eventSourceARN, sqsRecord);
+    } catch (error) {
+      // If error is ExecutionAlreadyExists, do not include in batchItemFailures
+      if (error.code === 'ExecutionAlreadyExists') {
+        logger.debug(`Execution already exists. Error: ${error}`);
+        return batchItemFailures;
```

Comment:
I think this can be removed.
Also, can you log the error for non-ExecutionAlreadyExists error case?

Reply:
If I don't return, then the `ExecutionAlreadyExists` error will be included in `batchItemFailures` list and I think it should be excluded. I can add logging.

Reply:
https://github.com/nasa/cumulus/pull/3389/commits/e395b00c215023cebba918d80722d12673da96bd

Reply:
It's confusing to `return batchItemFailures` here.  You can add `else` statement

Reply:
This is the message from sqs2sf:
`"message": "Execution already exists. Error: ExecutionAlreadyExists: Execution Already Exists: 'arn:aws:states:us-east-1:596205514787:execution:passTestSf_1686876287637:startSfSpec_12345'",`

Suggest to update to warning to avoid concerns, and remove redundancy:
logger.debug(`Warning: ${error}`);